files/_assets.conf.erb

```diff
@@ -38,4 +27,15 @@
     ExpiresActive On
     ExpiresDefault "access plus 1 year"
   </IfModule>
+
+  # Return compressed assets if they are precompiled
+  <IfModule mod_rewrite.c>
```

Comment:
I think the `IfModule` part mostly stems from when we used to support dropping in some file, but not really managing Apache. Any thoughts on dropping it and adding the rewrite rules unconditionally?

Reply:
We use it unconditionally in other places already, so ack.